README.md

```diff
@@ -41,6 +41,9 @@ And configure automatic Prometheus discovery:
         prometheus.io/scrape: "true"
         prometheus.io/port: "8080"
 ```
+
```

Comment:
Thank you, looks much better now.
The last minor nit: could you please also cleanup this whitespace diff?

Reply:
hope, it is now fine.